tests/test_orchestration/test_orchestrator.py

```
@@ -69,6 +67,121 @@ def test_registers_background_eval_log_as_artifact(self, tmp_path):
         assert tmp_path / "nel_eval.log" in artifact_files
 
 
+class TestRunSingleBenchmarkCleanup:
+    def _bench(self):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_bench`

(ANN202)